packages/api/src/graphql/permissions.ts

```diff
@@ -483,6 +483,30 @@ export const CanUpdateSettings: Permission = {
   deprecated: false
 }
 
+export const CanCreateTag: Permission = {
+  id: 'CAN_CREATE_TAG',
+  description: 'Allows to create a tag',
+  deprecated: false
+}
+
+export const CanUpdateTag: Permission = {
+  id: 'CAN_UPDATE_TAG',
+  description: 'Allows to update a tag',
+  deprecated: false
+}
+
+export const CanGetTag: Permission = {
+  id: 'CAN_GET_TAG',
```

Comment:
Should probably be "CAN_GET_TAGS", so plural. In terms of consistency with other permissions.